pints/tests/shared.py

```diff
@@ -375,3 +375,23 @@ def n_parameters(self):
         """ See :meth:`pints.Boundaries.n_parameters()`. """
         return self._n_parameters
 
+
+class SwappingTransformation(pints.Transformation):
+    """
+    Transformation that swaps the parameters around (as a very simple example
+    of a transformation that isn't elementwise).
+    """
```

Comment:
Is this useful to have??

Reply:
It's only for testing! I needed a non-elementwise transformation